Cargo.toml

```diff
@@ -1,6 +1,15 @@
 [workspace]
-members = [ "ingest", "db_store", "file_store", "mobile_rewards", "status", "mobile_verifier", "entropy",
-"lora_verifier", "poc_iot_injector"]
+members = [
+    "ingest",
+    "db_store",
+    "file_store",
+    "mobile_rewards",
+    "status",
+    "mobile_verifier",
+    "entropy",
+    "lora_verifier",
+    "poc_iot_injector",
```

Comment:
Missing `density_scaler`?

Reply:
That’s my assumption since cargo can’t find my test there. I had thought this was intentional since the metrics support crate isn’t listed either and it is likewise only run in the context of other crate services but I’m guessing that was just an oversight i cargo-culted